solidity/contracts/token/TokenBridgeCctpBase.sol

```diff
@@ -236,91 +235,13 @@ contract TokenBridgeCctp is
         bytes calldata /*metadata*/,
```

Comment:
Maybe limit this function to the mailbox? I see no issue, but if the mailbox is the only intended caller, then adding a check seems prudent.